Apps/Patient/src/Controllers/PatientController.cs

```diff
@@ -59,11 +65,34 @@ public PatientController(IPatientService patientService)
         /// </response>
         [HttpGet]
         [Produces("application/json")]
+        [ApiVersion("1.0")]
         [Route("{hdid}")]
         [Authorize(Policy = PatientPolicy.Read)]
         public async Task<RequestResult<PatientModel>> GetPatient(string hdid)
         {
             return await this.service.GetPatient(hdid).ConfigureAwait(true);
         }
+
+        /// <summary>
+        /// Gets a json of patient record.
+        /// </summary>
+        /// <param name="hdid">The patient hdid.</param>
+        /// <returns>The patient record.</returns>
+        /// <response code="200">Returns the patient record.</response>
+        /// <response code="404">The patient could not be found.</response>
+        /// <response code="401">The client must authenticate itself to get the requested response.</response>
+        /// <response code="403">
+        /// The client does not have access rights to the content; that is, it is unauthorized, so the server
+        /// is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.
+        /// </response>
+        [HttpGet]
+        [Produces("application/json")]
+        [ApiVersion("2.0")]
+        [Route("{hdid}")]
+        [Authorize(Policy = PatientPolicy.Read)]
+        public async Task<OkObjectResult> GetPatientV1(string hdid)
+        {
+            return this.Ok(await this.serviceV2.GetPatient(hdid).ConfigureAwait(true));
+        }
```

Comment:
Based on [this documentation](https://learn.microsoft.com/en-us/aspnet/core/web-api/action-return-types?view=aspnetcore-6.0#iactionresult-type-1), maybe we should return `Task<IActionResult>` and decorate the method with `[ProducesResponseType]` annotations.

Reply:
ok